packages/teleport/src/Databases/ConnectDialog/ConnectDialog.tsx

```diff
@@ -37,10 +37,18 @@ export default function ConnectDialog({
   const { hostname, port } = window.document.location;
   const host = `${hostname}:${port || '443'}`;
 
-  const connectCmd =
-    authType === 'sso'
-      ? `tsh login --proxy=${host}`
-      : `tsh login --proxy=${host} --auth=local --user=${username}`;
+  let connectCmd;
+  switch (authType) {
+    case 'sso':
+      connectCmd = `tsh login --proxy=${host}`;
+      break;
+    case 'local':
+      connectCmd = `tsh login --proxy=${host} --auth=local --user=${username}`;
+      break;
+    case 'passwordless':
+      connectCmd = `tsh login --proxy=${host} --auth=passwordless --user=${username}`;
```

Comment:
I did a double check to confirm that the `user` flag works together with `--auth=passwordless` and it does.

If I don't pass the user flag and I have multiple passwordless MFAs set up, tsh will ask me to pick one of them. If I pass the user flag, it won't ask and automatically log in as that user.